### PR TITLE
Fix chapters

### PR DESCRIPTION
**Pull request by vilem-k**
- Fix chapter ConnectMe: https://3.basecamp.com/5435868/buckets/29206288/todos/6071163510
- Fix chapter Architecture: https://3.basecamp.com/5435868/buckets/29206288/todos/6060610421

### CODE REVIEW

File: docs/pages/1_user/02_connect_me.md [hunk 1]
```diff
@@ -74,10 +76,14 @@ The following steps describe how to download and install the Shieldoo desktop cl
...
+This section describes establishing a secure connection to a server and attached services in Shieldoo Secure Network:
+1. [Connect to Servers](/connect_me/#connect-to-servers)
+2. [Connect to the Attached Services](/connect_me/#connect-to-the-attached-services)
 
-- __Servers__ are devices to which users or other servers can connect.
-- __Attached Services__ are a device that runs on the local network of a server but on which Shieldoo Secure Network cannot be installed (e.g. a printer). Shieldoo lets you directly connect to such services without connecting to their servers. These services are managed by the admin of Shieldoo Secure Network.
+{: .note }
+>__Servers__ are devices to which users or other servers can connect.
+>
+>__Attached Services__ are a device that runs on the local network of a server but on which Shieldoo Secure Network cannot be installed (e.g. a printer). Shieldoo lets you directly connect to such services without connecting to their servers. These services are managed by the admin of Shieldoo Secure Network.
```
Comment (pavel-john) [inline docs/pages/1_user/02_connect_me.md:86]: Tady mas "Attached Services are" - cili mnozne cislo, ale pak "a device that runs..." cili jednotne. Chtelo by to sladit.

File: docs/pages/2_admin/05_architecture.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The Shieldoo web application allows you to interact with the network architectur
 1. Log in to your organization's Shieldoo Secure Network web application.
 2. Enable __Expert Mode__
   - See the [Access Management](/access_management/) chapter to learn how to enable Expert Mode.
-3. Go to the __Setting & Logs__ \| __System Configuration__ section.
+3. Go to the __Setting & Logs__ \| __System Settings__ section.
```
Comment (pavel-john) [inline docs/pages/2_admin/05_architecture.md:27]: Tohle je taky divny, ale mozna je to i v Appce "Setting & Logs".. to by chtelo asi taky "Settings", obzvlast kdyz tam je "System Settnigs"

**Review by pavel-john — APPROVED** [review]

File: docs/pages/1_user/02_connect_me.md (hunk 1 above)
Comment (vilem-k) [reply docs/pages/1_user/02_connect_me.md:86]: Máš pravdu. Opravím to.

File: docs/pages/2_admin/05_architecture.md (hunk 2 above)
Comment (vilem-k) [reply docs/pages/2_admin/05_architecture.md:27]: Nene, v appce je "Settings & Logs". Opravím to.

**Pull request merged**